### DISCUSSION TITLE
Integration test for jobs

### DISCUSSION THREAD

**Discussion by fikretasma** [quartznet/quartznet#2269] [Q&A] [answered]
Hi . How can I write integration test for quartz.net jobs to test if the jobs did actually worked as excepted ? I searched for examples but couldn't find any example for the integration test.

**Comment by lahma** [comment, marked as answer]
You would test it like any other code you've written. `var job = new MyJob(); await job.Execute(context); // checks for outcome that it's desired` . You can think this like if you job's `Execute` method's only line of code would be `new MyService().DoSomething();` - how would you test your `MyService`?
